client/stylesheets/_modal.scss

```diff
@@ -386,6 +386,71 @@
 
 /*~~ Button End ~~*/
 
+/*~~~~ Received Offers Main ~~~~*/
+
+.profileReceivedOffersLi:hover {
+	background-color: #d3d3d3 !important;
+}
+
+.profileReceivedOffersH2 {
+	width: 76%;
+	margin: 40px auto 30px auto !important;
+}
+.profileReceivedOffersH2 a {
```

Comment:
Merge rule `.profileReceivedOffersH2 a` with rule on line 395<br>Selector `profileReceivedOffersH2` should be written in lowercase with hyphens